crates/hdk/src/prelude.rs

```diff
@@ -0,0 +1,46 @@
+
+/// Types required by all but the most trivial zomes. 
+/// This can greatly simplify imports for the majority of developers
+/// by simply adding use hdk::prelude::*;
+
```

Comment:
This is really the only file needed to be reviewed. Please check if you think these types are sufficient or if they are too broad. I made a judgement call to include these ones in particular based on the app spec.